products/api-platform.md

```diff
@@ -19,48 +19,56 @@ releases:
 #    eol: false
 #    latest: "2.7.0"
 
-  - releaseCycle: "2.6"
+-   releaseCycle: "2.6"
     release: 2021-01-22
     support: true
     eol: false
     latest: "2.6.8"
 
-  - releaseCycle: "2.5"
+    latestReleaseDate: 2022-01-11
+-   releaseCycle: "2.5"
     release: 2019-09-30
     support: 2021-01-22
     eol: false
-    latest: "2.5.7"
+    latest: "2.5.10"
 
-  - releaseCycle: "2.4"
+    latestReleaseDate: 2021-01-22
+-   releaseCycle: "2.4"
     release: 2019-03-22
     support: 2019-09-30
     eol: 2021-01-22
     latest: "2.4.7"
 
-  - releaseCycle: "2.3"
+    latestReleaseDate: 2019-09-17
+-   releaseCycle: "2.3"
     release: 2018-07-06
     support: 2019-03-22
     eol: 2019-09-30
     latest: "2.3.6"
 
-  - releaseCycle: "2.2"
+    latestReleaseDate: 2019-01-15
+-   releaseCycle: "2.2"
     release: 2018-02-16
     support: 2018-07-06
     eol: 2019-03-22
-    latest: "2.2.8"
+    latest: "2.2.10"
 
-  - releaseCycle: "2.1"
+    latestReleaseDate: 2019-01-15
+-   releaseCycle: "2.1"
     release: 2017-09-08
     support: 2018-02-16
     eol: 2018-07-06
     latest: "2.1.6"
 
-  - releaseCycle: "2.0"
-    release: 2017-01-06
+    latestReleaseDate: 2018-02-12
+-   releaseCycle: "2.0"
+    release: 2016-11-24
```

Comment:
https://github.com/api-platform/core/tags?after=v2.1.0-beta.1

Accurate, tag was created+signed on 2016-11-24, but the GitHub release was created a bit later. https://github.com/api-platform/core/releases/tag/v2.0.0